problems/3.conditionals.js

```diff
@@ -2,25 +2,40 @@
 * @param {number} temp - The current temperature
 * @returns {string} - A message about the temperature
 */
-function temperatureMessage() {
+function temperatureMessage(temp) {
+  let temp = 32
```

Comment:
YOu shouldn't be giving temp a value. Whomever calls the function will pass in the temp. You setting it to 32 is permanent and won't allow the user to have any input.